src/topology/quasi_separated.lean

```diff
@@ -46,15 +46,19 @@ class quasi_separated_space (α : Type*) [topological_space α] : Prop :=
 (inter_is_compact : ∀ (U V : set α),
   is_open U → is_compact U → is_open V → is_compact V → is_compact (U ∩ V))
 
-lemma is_quasi_separated_univ (α : Type*) [topological_space α] :
+lemma is_quasi_separated_univ_iff (α : Type*) [topological_space α] :
```

Comment:
Because this is an iff, `α` should be implicit. And actually `univ` is enough to figure `α` out, so you should also make it implicit in `is_quasi_separated_univ` (that matches `is_compact_univ`).
```suggestion
lemma is_quasi_separated_univ_iff :
```